src/http/http-axios-impl.ts

```diff
@@ -35,7 +35,9 @@ export class HttpAxiosImpl extends BaseHttp implements Http {
       },
       transformResponse: (data, headers) =>
         this.serializer.deserialize(
-          headers['Content-Type'] ?? 'application/json',
+          (headers['Content-Type'] ?? 'application/json')
+            .get('Content-Type')
+            .replace(/\s*;.*$/, ""),
```

Comment:
I'd like to avoid repeating this code in both `axios-impl` and `native-impl` because I'm going to keep maintaining both implementations until `fetch` is well supported on Node.js. 

How about adding something like `getContentType(header): string` to [`BaseHttp`](https://github.com/neet/masto.js/blob/main/src/http/base-http.ts) class so that they can call the inherited method?

Reply:
That sounds way better.  I'm going to extract duplicated code into a common method.